packages/manager/modules/dbaas-logs/src/logs/translations/Messages_fr_FR.json

```
@@ -486,6 +486,12 @@
   "inputs_state_running": "Opérationnel",
   "inputs_state_unknown": "Inconnu",
   "inputs_nb_instance": "Instances",
+  "inputs_autoscaled": "Mise à l'échelle automatique",
+  "inputs_autoscaled_detailled": "Mise à l'échelle automatique du nombre d'instance",
+  "inputs_autoscaled_explanation": "Le nombre d'instances de votre outil de collecte peut être soit fixe, soit mis à l'échelle automatiquement en fonction de la charge qu'il a à absorber.",
```

Maybe the sentence can be simplified ? 

```suggestion
  "inputs_autoscaled_explanation": "Le nombre d'instances de votre outil de collecte peut être soit fixe, soit mis à l'échelle automatiquement en fonction de la charge à absorber.",
```

---

You are right, applied